frontend/packages/ux-editor/src/components/config/componentSpecificContent/Summary2/Override/Summary2OverrideEntry.tsx

```diff
@@ -56,14 +58,33 @@ export const Summary2OverrideEntry = ({
     );
   }
 
+  const defaultOverrideConfig = (
+    componentType: ComponentType,
+  ): Omit<Summary2OverrideConfig, 'componentId'> => {
+    switch (componentType) {
+      case ComponentType.RepeatingGroup:
+        return { display: 'full' };
+      case ComponentType.Subform:
+        return { display: 'table' };
+      default:
+        return {};
+    }
+  };
```

Comment:
```suggestion
  const defaultOverrideConfig = (
    componentType: ComponentType,
  ): Omit<Summary2OverrideConfig, 'componentId'> => {
    switch (componentType) {
      case ComponentType.RepeatingGroup:
        return { display: 'full' };
      case ComponentType.Subform:
        return { display: 'table' };
      case ComponentType.MultipleSelect
      case ComponentType.Checkboxes
        return { displayType: 'list' };
      default:
        return {};
    }
  };
```

Very nicely done setting the default! 💪
What do you think about extending this logic to also set defaults for Checkboxes and MultipleSelect? That way, we could remove the `|| 'list'` from the `value={displayType || 'list'}` logic in `Summary2OverrideDisplayType.tsx` and maintain a consistent default setting for both display and displayType.

Reply:
Great suggestion! 👍
Adding a stacked PR implementing this

Reply:
pushed: https://github.com/Altinn/altinn-studio/pull/14654